dvc/repo/experiments/init.py

```diff
@@ -200,21 +199,19 @@ def init(
     defaults = defaults.copy() if defaults else {}
     overrides = overrides.copy() if overrides else {}
 
-    with_live = type == "dl"
+    with_live = "live" in overrides
 
     if interactive:
         defaults = init_interactive(
             validator=partial(validate_prompts, repo),
             defaults=defaults,
-            live=with_live,
             provided=overrides,
+            live=with_live,
             stream=stream,
         )
     else:
         if with_live:
```

Comment:
If we go with https://github.com/iterative/dvc/pull/7703/files#r867850140, we will only have a single place where that variable is used. :)
```suggestion
        if "live" in overrides:
```